test/unit/test_utils.lua

```diff
@@ -1,25 +1,32 @@
 local utils = require("kong.plugins.oidc.utils")
 local lu = require("luaunit")
+local opts_fixture = nil
```

Comment:
leave a comment here to indicate that you changed the options to be global to avoid possible mutation for the new test